restaurants/src/app/components/restaurantdetails/restaurantdetails.component.html

```diff
@@ -0,0 +1,13 @@
+<div *ngIf="restaurant">
+    <h2>{{restaurant.name | uppercase}} Details</h2>
+    <div>
+      <label for="restaurant-name">Restaurant name: </label> 
+        {{restaurant.name}}
+    </div>
+    <div><span>rank: </span>{{restaurant.rank}}</div>
```

Comment:
Same comment from above applies to this chunk of code.

Reply:
done